src/modules/ExistingCard.vue

```diff
@@ -22,12 +24,32 @@
 <script lang="ts">
 import { defineComponent } from 'vue'
 import CardsList from 'src/modules/CardsList.vue'
+import { ProductItemProps } from 'src/models/Tabs'
+import ProductEnum from 'src/models/enums/ProductEnum'
 
 export default defineComponent({
   name: 'ExistingCard',
   components: {
     CardsList,
   },
+  props: {
+    product: {
+      type: Object as () => ProductItemProps,
+      required: true,
+    },
+  },
+  methods: {
+    getPaymentOption(): string {
+      switch (this.product.productType) {
+        case ProductEnum.LongTermInterestFree:
+          return 'Long Term Finance'
+        case ProductEnum.PaymentHoliday:
+          return 'Payment Holiday'
+        default:
+          return this.product.productType
+      }
```

Comment:
design:
![image](https://user-images.githubusercontent.com/63985172/140861203-200d85ed-6c3b-4c38-8aa1-2e0909b2f4ff.png)
and
![image](https://user-images.githubusercontent.com/63985172/140861218-5376bccc-7afe-4150-98f2-bddf97c6f99f.png)